ui/imports/utils/Utils.qml

```diff
@@ -301,8 +301,8 @@ QtObject {
         const formatDDMMYY = "dddd d MMMM yyyy"
         const formatMMDDYY = "dddd, MMMM d, yyyy"
         const currentFormat = isDDMMYYDateFormat ? formatDDMMYY : formatMMDDYY
-        return !!value ? Qt.formatDate(new Date(), currentFormat) :
-                         Qt.formatDate(new Date(value), currentFormat)
+        return (value === undefined) ? Qt.formatDate(new Date(), currentFormat) :
```

Comment:
Oh it was reversed before. Maybe we could just reverse it back so it supports also `null` and `""`?
```suggestion
        return !value ? Qt.formatDate(new Date(), currentFormat) :
```